### ISSUE TITLE
Build failure: cuda-toolkit for ollama

### ISSUE THREAD

**Issue by luizribeiro** [NixOS/nixpkgs#305583]
### Steps To Reproduce

1. build `ollama` from unstable with `(unstable.ollama.override { acceleration = "cuda"; })`

### Build log

```
error: builder for '/nix/store/g9dda5xggh2lpz464d1fsyf6285l5b3i-cuda-toolkit.drv' failed with exit code 1;
       last 3 log lines:
       > warning: collision between `/nix/store/3csipjn30fjsa4lmxj68pb7wp1cbg6yn-cuda_cudart-12.2.140/LICENSE' and `/nix/store/hqvwymsafsxvrnlwn7jyhdmjj4m5c9c6-cuda-merged-12.2/LICENSE'
       > created 391 symlinks in user environment
       > rm: cannot remove '/nix/store/qlm47mx7h7r31l4gwrzimb1bik38k3y4-cuda-toolkit/lib64': No such file or directory
       For full logs, run 'nix log /nix/store/g9dda5xggh2lpz464d1fsyf6285l5b3i-cuda-toolkit.drv'.
error: 1 dependencies of derivation '/nix/store/jyra74n2xicnc4mf112hwl7iaj387k21-ollama-0.1.31.drv' failed to build
error: 1 dependencies of derivation '/nix/store/q2630sbkw2ibcs20qnfi76wlcd64xj7v-system-path.drv' failed to build
```

### Additional context

Perhaps this is related to the fix on #303046 ?

### Notify maintainers

@drupol @abysssol 

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
luiz@sodium ops % nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.8.5, NixOS, 23.11 (Tapir), 23.11.20240410.b2cf36f`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - channels(root): `"nixos-21.11.335443.e84444b14cc"`
 - channels(luiz): `""`
 - nixpkgs: `/nix/store/bwibdavbf6p8lci025h3qmln6laq5x82-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by abysssol** [comment]
This issue is related to #304924, in that it's fixed by the same pull request, #304736, which seems to have made it into an unstable release. You should be able to just update your unstable channel or flake input to get the fix.

**Comment by mevatron** [comment]
@abysssol Thanks I tested again today, and apparently the update just took a while to roll from master -> unstable. Thanks again for maintaining the ollama package!
